content/en/user-guide/ci/codebuild/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "CodeBuild"
+linkTitle: "CodeBuild"
+weight: 5
+description: Use LocalStack in CodeBuild
+---
+
+## Introduction
+
+CodeBuild is a managed AWS service for the build and testing phases of software development. CodeBuild allows you to define your build project, set the source code location, and handles the building and testing, while supporting various programming languages, build tools, and runtime environments. LocalStack supports CodeBuild out of the box and can be easily integrated into your pipeline to run your tests against a cloud emulator.
+
+## Getting started
+
+This guide is designed for users new to CodeBuild and assumes basic knowledge of YAML and LocalStack tooling. In this guide, we will show you how to set up a CodeBuild project that uses LocalStack to run basic S3 tests against it.
+
+### Configure your CodeBuild project
+
+Navigate to the AWS Management Console and open the CodeBuild console at [https://console.aws.amazon.com/codesuite/codebuild/home](https://console.aws.amazon.com/codesuite/codebuild/home).
+
+Click on **Create build project**. In the **Create build project** page, enter the following details:
+
+- **Project name**: Enter a name for your build project.
+- **Source**: Select the source code provider and repository details. For this example, we will use the the **No Source** option.
+- **Operating system**: Select the operating system for your build environment. For this example, we will use **Ubuntu**.
+- **Runtime**: Select the runtime for your build environment. For this example, we will use **Standard**.
+- **Image**: Select the image for your build environment. For this example, we will use **aws/codebuild/standard:7.0**.
+- **Service Role**: Select the service role for your build environment. For this example, we will use the **New service role** option.
```

Comment:
I would remove the sentences and just keep it simple. Mention this setup is for the example and just keep the list simple.

Source: No Source
Operating System: Ubuntu
... and so on

Reply:
For completeness, I would be in favor of keeping them as pointers and bold highlighting to mark the UI elements

Reply:
If you believe it is more comprehensible, then ok.